source/Preferences.cpp

```diff
@@ -54,6 +54,9 @@ namespace {
 	const vector<string> VSYNC_SETTINGS = {"off", "on", "adaptive"};
 	int vsyncIndex = 1;
 
+	const vector<string> CAMERA_ACCELERATION_SETTINGS = {"off", "on", "reversed"};
+	int cameraAccelerationIndex = 1;
```

Comment:
This should probably be default off.
```suggestion
	int cameraAccelerationIndex = 0;
```